### PR TITLE
Print error if query execution fails

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

**Review by niladrem — APPROVED** [review]
<3

File: streaming_jupyter_integrations/magics.py [hunk 1]
```diff
@@ -380,11 +380,13 @@ def __interrupt_execute(self, *args: Any) -> None:
 
     def __handle_done(self, fut: Any) -> None:
         self.background_execution_in_progress = False
-        print("Execution done")
         # https://stackoverflow.com/questions/48161387/python-how-to-print-the-stacktrace-of-an-exception-object-without-a-currently
         # will raise an exception to the main thread
```
Comment (swtwsk) [inline streaming_jupyter_integrations/magics.py:384]:
```suggestion
        # https://stackoverflow.com/questions/48161387/python-how-to-print-the-stacktrace-of-an-exception-object-without-a-currently
```

Please remove the second line of that comment, as it is not true from now on :)

**Review by swtwsk — CHANGES_REQUESTED** [review]

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM  :+1:

File: streaming_jupyter_integrations/magics.py (hunk 1 above)
Comment (grzegorz8) [reply streaming_jupyter_integrations/magics.py]: done

**Review by swtwsk — APPROVED** [review]
LGTM 🚀

**Pull request merged**